site/docs/red-team/plugins/harmbench.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_label: HarmBench
+---
+
+# HarmBench Plugin
+
+The HarmBench plugin uses the [HarmBench dataset](https://github.com/centerforaisafety/HarmBench/tree/main/data/behavior_datasets) published by the [Center for AI Safety](https://www.safe.ai/) to test LLM responses to harmful prompts. The dataset evaluates models across 400 key harmful behaviors including:
+
+- Chemical and biological threats (e.g., dangerous substances, weapons)
+- Illegal activities (e.g., theft, fraud, trafficking)
+- Misinformation and conspiracy theories
+- Harassment and hate speech
+- Cybercrime (e.g., malware, system exploitation)
+- Copyright violations
+
+Unlike many of our other plugins, the HarmBench plugin uses a statically generated dataset vs dynamically generating cases such as those for [PII leak detection](pii.md), [excessive agency](excessive-agency.md), [hallucination](hallucination.md) and others. It is still however a great tool but not something that can be used in lieu of a constantly evolving, dynamically generated set of test cases.
```

Comment:
this too, although I would draw comparisons to our harmful collection which is partially based on the harmbench taxonomy and not these

Reply:
Updated this comment.